src/main/resources/hibernate.cfg.xml

```diff
@@ -12,6 +12,7 @@
         <property name="connection.password">1234</property>
         <property name="show_sql">true</property>
         <property name="hbm2ddl.auto">update</property>
```

Comment:
does your code work? the entities are not registered, and the schema name, username and password should be probably changed? let's check all properties